docs/configuration-ini.md

```diff
@@ -101,159 +98,395 @@ beta2 = 0.999
 epsilon = 1e-7
 ```
 
-### DataSet Section
-
-Define data set. training, validation, test data set.
-
-Start with "[ DataSet ]"
-
-1. ```buffersize = <unsigned int>```
-
-    Define Buffer size. usually it is greater than batch size.
-
-    Data buffer thread keeps reading the data from the file and stores the data into the data buffer.
-    Meanwhile main thread gets the training data from this data buffer and feeds it to the model.
-    This keyword defines the size of Data Buffer.
-
-2. ```traindata = <string>```
-
-    training data file path.   The data must be saved as following
-
-    ```feature data[i], label data[i], feature data[i+1], label data[i+1], ...```
+### Learning Rate Scheduler Section
 
-3. ```validdata = <string>```
+Define the learing rate, decay steps and decay rate.
 
-    validation data file path.   The data must be saved as following
-
-    ```feature data[i], label data[i], feature data[i+1], label data[i+1], ...```
-
-4. ```testdata = <string>```
-
-    test data file path.   The data must be saved as following
-
-    ```feature data[i], label data[i], feature data[i+1], label data[i+1], ...```
-
-5. ```labeldata = <string>```
-
-    label data file path. The data must be saved as following
-
-    ```class Name [i], class name [i+1],...```
-
-### Layer Section
+1. ```learning_rate = <float>```
 
-Describe hyper-parameters for layer. Order of layers in the model follows the order of definition of layers here from top to bottom.
-
-Start with "[ ${layer name} ]". This layer name must be unique throughout network model.
-
-1. ```type = <string>```
-
-   Type of Layer
-     * input : input layer
-     * conv2d : 2D convolution layer
-     * pooling2d : 2D pooling layer
-     * flatten : flatten layer
-     * fully_connected : fully connected layer
-     * batch_normalization : batch normalization layer
-     * activation : activation layer
-
-2. ```kernel_size = <unsigned int>,<unsigned int>```
-
-   Kernel size for convolution layer
-
-3. ```bias_init_zero = <bool>```
+   Initial learning rate to decay
 
-   token to initialize bias with zeros. Setting to False would initialize bias randomly.
+2. ```decay_steps = <float> ```
```

Comment:
Please describe that decay_steps, decay_rate is used in exponential scheduler.
And also can you please describe properties for step scheduler?

Reply:
okay